circle.yml

```diff
@@ -1,12 +1,34 @@
+general:
+  artifacts:
+  - ~/drafter/emcc/jsdrafter.js
+  - ~/drafter/emcc/jsdrafter.js.mem
```

Comment:
Is there somewhere to download these? Maybe we should link to them in the README?